var/spack/repos/builtin/packages/zlib/package.py

```diff
@@ -50,12 +50,30 @@ class Zlib(Package):
 
     patch('w_patch.patch', when="@1.2.11%cce")
 
+    @property
+    def headers(self):
+        # If zlib is configured as external package, e.g. in /usr, searching
```

Comment:
i have the same comment here: instead of fixing it in per-package basis, those things should go into the defaults.

Reply:
@adamjstewart @alalazo ping.